src/main/kotlin/org/jetbrains/research/testspark/display/TestCasePanel.kt

```diff
@@ -136,6 +140,14 @@ class TestCasePanelBuilder(
     private val initialCodes: MutableList<String> = mutableListOf()
     private val lastRunCodes: MutableList<String> = mutableListOf()
     private val currentCodes: MutableList<String> = mutableListOf()
+    private val trimmedCodeToError = mutableMapOf<String, String?>()
+
+    init {
+        prepareUpperPanel()
+        prepareMiddlePanel()
+        prepareBottomPanel()
+        updateErrorRelatedUI()
+    }
 
     /**
      * Retrieves the upper panel for the GUI.
```

Comment:
`Retrieves` -> `builds/assembles`, as the implementation does not retrieve the panel from anywhere, but populates it with UI elements.

Reply:
Same for the two other analogous methods

Reply:
it's an old explanation, I'll change it